optaplanner-examples/src/main/java/org/optaplanner/examples/common/experimental/ExperimentalConstraintCollectors.java

```diff
@@ -30,14 +30,19 @@
 import org.optaplanner.core.impl.score.stream.quad.DefaultQuadConstraintCollector;
 import org.optaplanner.core.impl.score.stream.tri.DefaultTriConstraintCollector;
 import org.optaplanner.core.impl.score.stream.uni.DefaultUniConstraintCollector;
+import org.optaplanner.examples.common.experimental.api.ConsecutiveInfo;
+import org.optaplanner.examples.common.experimental.impl.ConsecutiveDataImpl;
+import org.optaplanner.examples.common.experimental.impl.ConsecutiveIntervalDataImpl;
+import org.optaplanner.examples.common.experimental.impl.ConsecutiveSetTree;
+import org.optaplanner.examples.common.experimental.impl.IntervalTree;
 
 /**
  * A collection of experimental constraint collectors
  * subject to change in future versions.
  */
 public class ExperimentalConstraintCollectors {
     /**
-     * Creates a constraint collector that returns {@link ConsecutiveData} about the first
+     * Creates a constraint collector that returns {@link ConsecutiveDataImpl} about the first
```

Comment:
We don't want this. Public types must not be `Impl`s.

Reply:
Also, it seems this returns `ConsecutiveInfo`, not `ConsecutiveData`.

Reply:
ConsecutiveInfo is the api type of `ConsecutiveDataImpl`; javadoc got stale